packages/account/src/providers/transaction-summary/operations.ts

```diff
@@ -177,38 +178,6 @@ export function getReceiptsTransferOut(receipts: TransactionResultReceipt[]) {
   return getReceiptsByType<TransactionResultTransferOutReceipt>(receipts, ReceiptType.TransferOut);
 }
 
-/** @hidden */
-export function getContractTransferOperations({ receipts }: ReceiptParam): Operation[] {
```

Comment:
Can you please briefly explain the splitting of the functions in this file? Looks like you removed this function but added parts of it to another function & added a new function too. An overview in the PR description would be awesome, especially since this is a breaking change 🙂

Reply:
@Dhaiwat10 I removed `getContractTransferOperations` because I've removed `OperationName.contractTransfer`.

I've added a better description to the PR. Please let me know if it's not clear enough. And sorry for not adding this sooner 🙏 .

Reply:
No worries at all @Torres-ssf, thanks for the new detailed description